### DISCUSSION TITLE
Broaden genus of Information Bearing Entity

### DISCUSSION THREAD

**Discussion by nklsbckmnn** [CommonCoreOntology/CommonCoreOntologies#382] [Scope Questions]
I would like to put up for discussion a change of the definition of IBE that allows for object aggregates to be IBEs.

An ICE currently needs to generically depend on at least one IBE. An ICE can be concretized as a role. 

An object aggregate (e. g. an organization) (and also a fiat object part) can bear a role that concretizes an ICE and so can carry that ICE, but it can't be an IBE. That means that currently for every object aggregate that carries some ICE, there needs to exist an additional IBE object that carries that ICE.

The case I have in mind is that there could be an organization that is a legal person and bears some subtype of a legal role that concretizes some directive ICE.

I guess a possible counterargument to this proposal is that object aggregates do not add anything physical to their object parts so that it's not clear what is supposed to carry the information. Roles seem to not raise the same problem since they are externally grounded.

**Comment by rorudn** [comment]
Can you state this in a different way, I don't think I understand the request. Here's my current understanding: expand the extension of information bearing entities to include specifically dependent continuants because SDCs concretize information content entities. Using that understanding my response would be that neither SDCs nor IBEs concretize ICEs. IBEs are the carriers of ICEs.

**Comment by nklsbckmnn** [comment]
Sorry, I rephrased the request.

**Comment by mark-jensen** [comment]
Moving this to discussion.
